docker/builder/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:alpine3.9
+FROM golang:alpine3.10
```

Comment:
Cloud we just use `latest` tag, if no version-specified feature is actually needed?

Reply:
latest tag is based on buster, not alpine.